pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/DeviceDriver.java

```diff
@@ -30,7 +31,7 @@ public abstract class DeviceDriver extends AbstractService implements AutoClosea
     private static final String CREATE_SCOPE_KEY = "CREATE_SCOPE";
 
     public DeviceDriver(DeviceDriverConfig config) {
-        this.config = config;
+        this.config = Preconditions.checkNotNull(config, "deviceDriverConfig");
```

Comment:
What message would it print when null? with "deviceDriverConfig" as second param?

Reply:
Below is the output
```
Exception in thread "main" java.lang.NullPointerException: deviceDriverConfig
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:907)
	at io.pravega.sensor.collector.DeviceDriver.<init>(DeviceDriver.java:34)
	at io.pravega.sensor.collector.simple.SimpleDeviceDriver.<init>(SimpleDeviceDriver.java:56)
	at io.pravega.sensor.collector.accelerometer.AccelerometerDriver.<init>(AccelerometerDriver.java:44)
```